infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutSwitch.kt

```diff
@@ -46,6 +66,14 @@ data class LayoutSwitch(
 
     fun getJoint(number: JointNumber): LayoutSwitchJoint? = joints.find { j -> j.number == number }
 
+    @get:JsonIgnore
+    val presentationJoint
+        get() = joints.find { j -> j.type == SwitchJointType.MAIN }
```

Comment:
Tästä voisi ottaa `@JsonIgnore`:n pois. Frontilla on paljon paikkoja joissa tarvitaan presentationjointtia ja olisi todella kiva jos niiden löytämiseen siellä ei tarvitsisi hakea structureita ja tehdä song-and-dancea `.find()`:in kanssa

Reply:
Njoo no nyt siellä on noissa jointeissa itsessään kuitenkin tuo tieto olemassa eli voi vain hakea sen findilla joints-listasta (structurea ei tarvitse enää). Voisi tämänkin toki laittaa menemään, mutta se on suora duplikaattitieto koska se hakee vain listalta.